novawallet/Common/View/ChainAccount/ChainAccountView.swift

```diff
@@ -13,12 +13,20 @@ final class ChainAccountView: UIView {
         return view
     }()
 
-    let networkLabel: UILabel = {
-        let label = UILabel()
-        label.font = .regularSubheadline
-        label.textColor = R.color.colorTextPrimary()
-        return label
-    }()
+    let multiLabelView: MultiValueView = .create { view in
+        view.valueTop.apply(style: .regularSubhedlinePrimary)
+        view.valueTop.textAlignment = .left
+
+        view.valueBottom.apply(style: .caption1Secondary)
+        view.valueBottom.textAlignment = .left
+        view.valueBottom.numberOfLines = 0
+        view.valueBottom.isHidden
```

Comment:
Forgot to assign a value. The main drawback of that approach is that we should use bind method to set the text's because otherwise we may end up with unneeded offset due to the spacing between title and subtitle. So, let's check that we are using bind method in all places where ChainAccountView is used.

Reply:
Fixed Part with Value. To address the drawbacks associated with adding MultiValueView, we have hidden valueBottom to avoid dealing with unnecessary offsets.